### DISCUSSION TITLE
What is the function of "top10perTarget" or "top50perTarget" in runSCENIC_2_createRegulons?

### DISCUSSION THREAD

**Discussion by smilesw** [aertslab/SCENIC#283] [* General SCENIC questions]
Hello everyone
I have looked for the function of "top10perTarget" or "top50perTarget" in runSCENIC_2_createRegulons, but I cannot find the detailed explanation except for toy run in case of "top5perTarget". 

What is the function and difference between the followings. 

: runSCENIC_2_createRegulons(scenicOptions)
: runSCENIC_2_createRegulons(scenicOptions, coexMethod=c("top10perTarget"))
: runSCENIC_2_createRegulons(scenicOptions, coexMethod=c("top50perTarget"))

Thanks in advance.

**Comment by DiracZhu1998** [comment]
<img width="781" alt="Screenshot 2024-01-04 at 20 04 58" src="https://github.com/aertslab/SCENIC/assets/47734593/7bbe473c-2560-40be-869c-afe1716bc3a6">

**Reply by DiracZhu1998** [reply]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9867709/pdf/42003_2023_Article_4457.pdf

**Reply by DiracZhu1998** [reply]
top 5, 10 and 50 TFs for each target gene
